fennel/client_tests/branches/test_clone.py

```diff
@@ -0,0 +1,390 @@
+from datetime import datetime
+
+import pandas as pd
+import pytest
```

Comment:
I dont see any tests for extractors. If I clone B from A and change the extractor definition the features returned by B should be different from A for eg ?

Reply:
added